src/pages/_app.js

```diff
@@ -7,28 +7,24 @@ import Alert from "@/src/components/Alert";
 import useLocalStorageState from "use-local-storage-state";
 
 export default function App({ Component, pageProps }) {
-  const [transactions, setTransactions] = useLocalStorageState("transactions", {
-    defaultValue: transactionsData,
-  });
-  const [mode, setMode] = useState("default");
   const router = useRouter();
-  const [alert, setAlert] = useState({
-    isOpen: false,
-    alertMessage: "",
-  });
+  const [transactions, setTransactions] = useLocalStorageState("transactions", { defaultValue: transactionsData });
+  const [alert, setAlert] = useState({ isOpen: false, alertMessage: "" });
+  console.log(transactions);
 
+  //-------------Alert logic-------------
   function handleAlert(alertMessage) {
-    setAlert({ isOpen: true, alertMessage});
+    setAlert({ isOpen: true, alertMessage });
   }
 
   function handleAlertClose() {
     setAlert({ isOpen: false, message: "" });
```

Comment:
In line 12 the key for the message is ```alertMessage```not ```message```

Reply:
this is fine this way